cypress/integration/testmovies_spec.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference types="cypress" />
+
+describe("usertesting moviepage", () => {
+  it("test that filterfunction can sort by highest rating", () => {
+    cy.visit("/movies");
+    cy.get(".MuiOutlinedInput-root > #demo-simple-select-filter")
```

Comment:
this one gives an error 
![image](https://user-images.githubusercontent.com/77925373/168053977-f948b406-9c0e-4c55-bc83-5930995108f9.png)

Reply:
fixing that, when MUI changed, paths have to change. no biggie